tests/settings/SectionFillingTest.py

```diff
@@ -40,6 +41,7 @@ def setUp(self):
         self.log_printer = LogPrinter(ConsolePrinter())
         self.section = Section('test')
         self.section.append(Setting('key', 'val'))
+        self.test_section = Section('')
```

Comment:
you already have `self.section` above for this purpose ;)

Reply:
self.section won't cut it @Makman2 . if it is not Section(' ') then len(needed_settings) > 0 won't be satisfied and the log warning won't happen

Reply:
self.section doesn't trigger the warning